Header header
geometry_msgs/Point location
Float64 reward
String agent_id
